src/aead/aead.rs

```diff
@@ -24,6 +24,71 @@
 //!
 //! [AEAD]: http://www-cse.ucsd.edu/~mihir/papers/oem.html
 //! [`crypto.cipher.AEAD`]: https://golang.org/pkg/crypto/cipher/#AEAD
+//!
+//! # Examples
+//!
+//!```rust
+//!extern crate ring;
+//!
+//!use ring::aead::*;
+//!use ring::rand::SystemRandom;
```

Comment:
Oops, I missed this before. You need something like `use ring::rand::{SystemRandom, SecureRandom};` to avoid build failures.